### DISCUSSION TITLE
New Badge Announcing Achievements

### DISCUSSION THREAD

**Discussion by andae** [community/community#18245] [Other Feature Feedback, Questions, & Ideas] [answered]
Today a new badge announcing an achievement is rolling out, whether it looks cool on you or not!

**Comment by MrKrishnaAgarwal** [comment, marked as answer]
I got 2 Badges,
They look great
![image](https://user-images.githubusercontent.com/100597998/173040839-7d6312bd-d4f8-453e-9e37-7bc1312cfe11.png)

**Reply by andae** [reply]
Yeah, it looks great on you

**Reply by MrKrishnaAgarwal** [reply]
Thanks you too

**Reply by MrKrishnaAgarwal** [reply]
by creating pull request, by merging them and by answering discussions.

**Reply by mbondareff** [reply]
Nice

**Comment by arrazaq** [comment]
Doesn't look good to me, too many badges look congested. It would be nice there to have a setting to select certain achievements to hide/share this could make it look better.

**Reply by MrKrishnaAgarwal** [reply]
True

**Reply by andae** [reply]
Oh yeah, that seems better

**Comment by 1vanK** [comment]
Kindergarten. How do I remove achievements from my profile?

**Reply by MrKrishnaAgarwal** [reply]
Settings / Public Profile / Show Achievements on my profile, uncheck it.

**Reply by MrKrishnaAgarwal** [reply]
Who is Kindergarten here

**Reply by 1vanK** [reply]
https://avatars.githubusercontent.com/u/100597998?v=4

**Reply by MrKrishnaAgarwal** [reply]
Sir I request you to talk to me with respect

**Reply by 1vanK** [reply]
I don't want

**Reply by aryan1107** [reply]
@MrKrishnaAgarwal Can you help me with yolo badge?

**Reply by MrKrishnaAgarwal** [reply]
You already have it

**Comment by Sulagna-Dutta-Roy** [comment]
I got Pull Shark badge bronze one.

**Reply by HUMBLEGROUND** [reply]
me too!

**Comment by devcrafta** [comment]
We can find out more about Badge here.
https://github.com/Schweinepriester/github-profile-achievements

**Comment by DhanushNehru** [comment]
I got Pull Shark badge one

**Reply by dheerajrao20** [reply]
Yeah! Congo @DhanushNehru

**Comment by HUMBLEGROUND** [comment]
I got the Pull Shark badge and the YOLO badge.
How do I get the Galaxy Brain badge?

**Reply by dheerajrao20** [reply]
After answering **two discussions**, You will get **Galaxy Brain badge**

**Comment by MichaelDimmitt** [comment]
Great badges , Keep them coming!

**Comment by andae** [comment]
Yeah' enjoy your new shining badge!

**Comment by fertrevino** [comment]
The Mars and Artic badges looked cool and respectable. The new badges make little sense to me

**Comment by GausAlMunirTushar** [comment]
I got one badges

**Comment by epistax1s** [comment]
I have a badge. But getting "GalaxyBrain" is broken...

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
